bindata/network/openshift-sdn/alert-rules.yaml

```diff
@@ -39,40 +39,39 @@ spec:
       for: 1h
       labels:
         severity: warning
-    - alert: IPTableSyncSDNPod
+    - alert: NodeProxyApplySlow
```

Comment:
are these names referred to from anywhere else?

Reply:
Not internally. If a customer has silenced the alert, this silence will be undone.

@openshift/openshift-team-monitoring, what's the stability guarantee for alert names?

Reply:
AFAIK there are no guarantees.

Reply:
Yep, we do not provide any guarantees.